query-connector/src/app/layout.tsx

```diff
@@ -32,6 +36,10 @@ export default async function RootLayout({
       <body>
         <div className="application-container">
           <DataProvider runtimeConfig={runtimeConfig} session={session}>
+            <SessionTimeout
+              idleTimeMsec={IDLE_TIMEOUT_MSEC}
+              promptTimeMsec={PROMPT_TIMEOUT_MSEC}
+            />
```

Comment:
These two parameters have been set as receiving props to make unit testing easier.